tests/framework/interface.go

```diff
@@ -28,8 +28,15 @@ type testRunner interface {
 }
 
 type Cluster interface {
+	Members() []Member
+	Client() Client
 	Close() error
+}
+
+type Member interface {
 	Client() Client
+	Start() error
+	Stop()
```

Comment:
Is it possible that the e2e or integration somehow fail to stop a member? Does it make sense to return an error?

Reply:
sg